### DISCUSSION TITLE
committer: Tyler White

### DISCUSSION THREAD

**Discussion by cpcloud** [ibis-project/governance#8] [Committer] [answered]
I'd like to nominate Tyler White (@IndexSeek) for committer status.

Tyler has been engaged with the project for quite some time, and in various ways, including blog posts, bug reports, feature requests and code contributions.

More recently Tyler has been adding numerous API examples in our docstrings, which is something that users are always requesting more of :)

I think Tyler shows a strong commitment to the long-term success of the project and think he would be a great person to have as a committer.

**Comment by gforsyth** [comment]
Yep, that's a big +1 from me!

**Comment by IndexSeek** [comment, marked as answer]
Thank you so much for the nomination, @cpcloud! It's an honor. 

I would greatly enjoy continuing to work closer on the project and hope to keep pushing Ibis forward!

**Reply by cpcloud** [reply]
I invited you to the organization earlier this morning!

Welcome aboard!
